packages/Button/src/Button.styles.js

```diff
@@ -42,6 +42,10 @@ const commonStyles = `
     }
   }
 
+  [data-whatinput="mouse"] &[data-has-forced-focus="true"]:focus {
+    box-shadow: ${tokens.highlight.active.noBorder.boxShadow}; 
+  }
+
```

Comment:
i think this could be nested under line 30, and cleaned up a bit (the `:focus` is re-used`):

    [data-whatinput="mouse"] &:focus {
      box-shadow: ${tokens.highlight.active.noBorder.boxShadow}; 

      &:not([data-has-forced-focus="true"]) {
        box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.1);
        border-color: ${tokens.border.color};

        &[kind="minor"],
        &[kind="link"] {
          box-shadow: none;
          border-color: transparent;
        }

        &[kind="flat"] {
          box-shadow: none;
        }
      }
    }

Reply:
Sure! But I guess I'll still need `[data-has-forced-focus="true"]`, otherwise it might be overrided by rawbutton component, it has `[data-whatinput="mouse"] &:focus {  box-shadow: none; }`